packages/dev-container/src/electron-node/remote-container-connection-provider.ts

```
@@ -287,13 +318,25 @@ export class RemoteDockerContainerConnection implements RemoteConnection {
 
     async copy(localPath: string | Buffer | NodeJS.ReadableStream, remotePath: string): Promise<void> {
         const deferred = new Deferred<void>();
-        const process = exec(`docker cp -qa ${localPath.toString()} ${this.container.id}:${remotePath}`);
+        const dockerHost = process.env.DOCKER_HOST;
+        let remoteHost = '';
+        try {
+            if (dockerHost) {
+                const dockerHostURL = new URL(dockerHost);
+                if (dockerHostURL.protocol === 'http:' || dockerHostURL.protocol === 'https:') {
+                    dockerHostURL.protocol = 'tcp:';
+                }
+                remoteHost = '-H ' + dockerHostURL.href + ' ';
+            }
+        } catch (_) { }
```

why the empty catch here? I suggest to at least log some message here